assets/style.css

```diff
@@ -0,0 +1,781 @@
+}
+
+.weezy {
+  background-color: #fe4736;
+  padding: 10px 0;
+  text-align: center;
+}
+
+.latest-funding-title-links {
+  width: 99%;
+}
+
+.seed-funds {
+  background-color: #ffe8e5;
+  padding: 11px 0;
+  height: 60px;
+}
+
+.seed-funds p {
+  margin-left: 100px;
+}
+
+.seed-funds-div2,
+.seed-fund-details {
+  font-size: 13px;
+}
+
+.seed-funds-div2 p {
+  padding: 10px 0 0 0;
+}
+
+span img {
+  width: 37px;
+  height: 9px;
+}
+
+.funding-grid img {
+  width: 80px;
+  position: absolute;
+  top: 40px;
+  margin-left: 10px;
+}
+
+.latest-funding-links > div {
+  flex-direction: column;
+}
+
+.list-items span {
+  border-bottom: 1px solid rgb(148, 143, 143);
+  padding: 5px 0;
+}
+
+.latest-funding-links > div span {
+  padding: 5px 0;
+}
+
+/* MOST POPULAR SECTION */
+.most-popular-section {
+  margin-bottom: 100px;
+}
+
+.most-popular-image {
+  height: 200px;
+  width: 94%;
+  margin-top: 20px;
+}
+
+.hardfork-img {
+  width: 100%;
+  height: 400px;
+}
+
+.latest-img {
+  width: 70px;
+  height: 100%;
+}
+
+.hardfork-content {
+  bottom: 10px;
+  left: 10px;
+}
+
+.hardfork-sub-content > div h3 {
+  font-size: 14px;
+}
+
+.hardfork-sub-content > div p {
+  font-size: 13px;
+}
+
+.list-items a::before {
+  content: "•";
+  color: #fe4736;
+}
+
+.list-items a {
+  color: #555;
+}
+
+.list-items {
+  display: flex;
+  flex-direction: column;
+}
+
+/* FOOTER */
+footer {
+  width: 100%;
+  height: 250px;
+  text-align: center;
+}
+
+.footer-links {
+  background-color: #131313;
+  padding: 2.5em 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  max-width: 100%;
+  padding: 0 20px;
+  height: 100%;
+}
+
+.footer-links-social {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  margin-bottom: 15px;
+  vertical-align: baseline;
+}
+
+.footer-links-social,
+.footer-links-social li a,
+.footer-links-menu {
+  justify-content: center;
+}
+
+.footer-links-social li a {
+  box-sizing: border-box;
+  background-color: #333;
+  margin: auto 5px;
+  border-radius: 20px;
+  padding: 10px;
+}
+
+.footer-links-menu {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  box-sizing: border-box;
+}
+
+.footer-links-menu li a {
+  margin: 5px;
+  border: 0;
+  vertical-align: baseline;
+  box-sizing: border-box;
+  align-content: center;
+  flex-wrap: nowrap;
+}
+
+.footer-legal {
+  width: 100%;
+  height: 100px;
+  padding: 50px 0;
+  background-color: #000;
+  display: block;
+}
+
+/* ****** =====================================
+MEDIA QUERIES==========================******* */
+
+@media only screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  /* NAVIGATION */
+
+  .first-navbar {
+    background-color: #131313;
+  }
+
+  .split-links,
+  .second-nav-links,
+  .footer-links {
+    justify-content: space-between;
+  }
+
+  .nav-flex {
+    width: 100%;
+  }
+
+  .first-navbar ul li {
+    margin-right: 10px;
+  }
+
+  .first-navbar li a {
+    color: #555;
+  }
+
+  .news {
+    color: #fff !important;
+  }
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_